recipes/pylandstats/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pylandstats" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "801f8814e4484329462cff8f7db781ea4b097e2f08ef88ad61378b4a612b9223" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt"
+  skip: true  # [win and py2k]
```

Comment:
This seems to be failing on python 3 and windows, should the selector be `[win or py2k]`?

Reply:
Yes, my mistake. Thank you!